01-04-2023-version-control-your-retool-app.md

```diff
@@ -1,20 +1,47 @@
 ---
 
-Subject: "Make Retool Better with Version Control"
+Subject: "Make Retool Better with Database Version Control"
 Date: "2023-01-04"
 ICP: "VP ENG, CTO, Devs"
 Market: "Any company using Retool"
 
 ---
 
-Hi First Name,
+# First Email
+
+`Greeting` 
 
 As a Retool user, you know firsthand the power of low-code solutions for creating internal tools. But have you ever encountered the challenge of accidentally making changes to your production database? Maybe today you work off of a copy or a backup to avoid that problem. But how do you manage changes to your database?
```

Comment:
```suggestion
As a Retool user, you know the power of low-code solutions for creating internal tools. But have you ever accidentally changed your production database? Maybe today you work off of a copy or a backup to avoid that problem. But how do you manage changes to your database?
```